src/Raven.Server/Documents/Handlers/QueriesHandler.cs

```diff
@@ -201,7 +203,7 @@ private async Task<IndexQueryServerSide> GetIndexQuery(JsonOperationContext cont
         private async Task SuggestQuery(IndexQueryServerSide indexQuery, DocumentsOperationContext context, OperationCancelToken token)
         {
             var existingResultEtag = GetLongFromHeaders("If-None-Match");
-
+            var totalDocumentSize = default(long);
```

Comment:
why not `0L` instead `default(long)`? or just `long totalDocumentSize` - you're assigning the value to it using `out` anyway

Reply:
done - must've missed that one